src/common/tracer.h

```diff
@@ -1,32 +1,95 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
-#ifndef TRACER_H_
-#define TRACER_H_
+#ifndef TRACER_H
+#define TRACER_H
+
+#ifdef HAVE_JAEGER
```

Comment:
this `HAVE_JAEGER` define comes from cmake, which processes `include/config-h.in.cmake` to generate `include/acconfig.h`. so your header needs to `#include "include/acconfig.h"` before this `#ifdef`